packages/ai-jsx/src/core/completion.tsx

```diff
@@ -236,6 +275,8 @@ export function Completion(
  *      <SystemMessage>You are a nice person.</SystemMessage>
  *      {/* This is fine, because MyUserMessage renders to a UserMessage. *}
```

Comment:
```suggestion
 *      {/* This is fine, because MyUserMessage renders to a UserMessage. */}
```

Whoops – now that we're rendering the docs, we see that this comment was missing a closing /.